database/migrations/2023_11_10_183002_replace_approved_column_with_status_in_engage_purchase_requests_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('engage_purchase_requests', static function (Blueprint $table): void {
+            $table->string('status');
```

Comment:
Do we need to map existing data's approved column? or can you just resync and get the data back?

Related: do you need to set a default value or make the column nullable here?

Reply:
Sync process correctly backfills the empty column.

I am deliberately not making it nullable or setting a default value because this field should always be set.